client/blocks/app-promo/README.md

```diff
@@ -0,0 +1,13 @@
+# App Promo
+
+This component is used to display the Mobile App Promo to users.
+
+## How to use
+
+```js
+import AppPromo from 'calypso/blocks/app-promo';
+
+function render() {
+	return <AppPromo />;
+}
+```
```

Comment:
I guess this isn't too useful, but I don't mind keeping it.

Reply:
I think the example is not too useful to you :P but if mobile folks have to make the changes having even simple examples and readme is probably a good thing. Lets leave it in for now.